src/apps/invitations/test_invite.py

```diff
@@ -857,12 +857,13 @@ async def test_shell_invite(self, client, session, shell_create_data, bob: User,
         subject = response.json()["result"]
 
         url = self.shell_acc_invite_url.format(applet_id=applet_id)
-        response = await client.post(url, dict(subjectId=subject["id"], email=email))
+        response = await client.post(url, dict(subjectId=subject["id"], email=email, language="fr"))
         assert response.status_code == http.HTTPStatus.OK
         assert len(TestMail.mails) == 1
         subject_model = await SubjectsCrud(session).get_by_id(subject["id"])
         assert subject_model
         assert subject_model.email == email
+        assert subject_model.language == "fr"
```

Comment:
Since this change is adding `language` as an optional parameter, could this be added as a second test instead?

Reply:
Added new test. thanks